app/domain/output_document.rb

```diff
@@ -5,7 +5,9 @@ class OutputDocument
 
   delegate :id, :income_in_retirement, :continue_working, :unsure,
```

Comment:
Good catch, I'm sure I'd removed them before but then I went crazy with the rebasing. I've changed the delegated attributes to just the id and the supplementary topics and everything's green.